pkg/plugin/plugin_test.go

```diff
@@ -2,35 +2,52 @@ package plugin
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"reflect"
 	"testing"
 
 	"github.com/golang/mock/gomock"
 	"github.com/sirupsen/logrus"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
 
 	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/test/util/tls"
+
 	"github.com/openshift/openshift-azure/pkg/util/mocks/mock_arm"
 	"github.com/openshift/openshift-azure/pkg/util/mocks/mock_cluster"
 	"github.com/openshift/openshift-azure/test/util/populate"
 )
 
 func TestMerge(t *testing.T) {
+	logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
+	entry := logrus.NewEntry(logrus.StandardLogger())
```

Comment:
please remove the line above and do `log := logrus.NewEntry(logrus.StandardLogger())` here